app/lib/analyzer.dart

```diff
@@ -0,0 +1,171 @@
+// Copyright (c) 2017, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+library pub_dartlang_org.analysis;
```

Comment:
Maybe use a consistent name for the filename and library name (i.e. analysis -> analyzer).